Package.swift

```diff
@@ -18,8 +18,8 @@ let package = Package(
     dependencies: [
         // A revision after Xcodeproj module is removed
         // https://github.com/apple/swift-package-manager/pull/5748
-        .package(url: "https://github.com/apple/swift-package-manager.git",
-                 revision: "swift-DEVELOPMENT-SNAPSHOT-2022-11-12-a"),
+        .package(url: "https://github.com/giginet/swift-package-manager.git",
```

Comment:
I'm using a fork to call a hidden implementation.

I submitted PR to publish this. However, this will not be accepted.
https://github.com/apple/swift-package-manager/pull/6049